### DISCUSSION TITLE
Looking for Feedback on this Repository

### DISCUSSION THREAD

**Discussion by dylanyoung-dev** [dylanyoung-dev/sitecore-cdp-serializer#17] [Polls]
Hello everyone! We are looking for feedback on how useful this repository is or will be for your organization.  Please vote and if you have more information you would like to share, leave your opinions below.

**Comment by mhak** [comment]
This is a game changer and definitely mandatory for any Sitecore cdp/personalize solutions at scale!

This addresses 2 issues I've been wondering about:

1. How do we source control CDP code / components?
2. In a typical two tenant scenario (stg and prod), how do we promote or transfer components and configurations from one environment to another?
